.github/workflows/lint.yml

```diff
@@ -29,21 +29,30 @@ jobs:
     steps:
       - name: Checkout source code
         uses: actions/checkout@v2
+      - name: Check PHP files diff
+        uses: technote-space/get-diff-action@v4
+        with:
+          PATTERNS: |
+            **/*.php
       - name: Setup PHP 7.4
+        if: ${{ env.GIT_DIFF }}
```

Comment:
Try to find or create an action that sets the test as passed and stops it?

Reply:
Done